### PR TITLE
Exclude SlevomatCodingStandard.ControlStructures.EarlyExit.UselessElseIf

### PR DESCRIPTION
**Pull request by malarzm**
I'm not entirely sure what side effects does it bring, but will fix something me and @alcaeus would like to see fixed :) Cross referencing https://github.com/doctrine/mongodb-odm/pull/1757

### CODE REVIEW

**Comment by lcobucci** [comment]
Can you provide a bit more information on why you prefer to allow that? I honestly find it more confusing

File: tests/input/EarlyReturn.php [hunk 1]
```diff
@@ -49,4 +51,17 @@ public function quoox() : bool
 
         return true;
     }
+
+    public function qux() : int
+    {
+        if (is_numeric($var)) {
+            if ($var > 0) {
+                return 1;
+            } elseif ($var < 0) {
```
Comment (Majkl578) [inline tests/input/EarlyReturn.php]: This is definitely useless `elseif`, it should be `if`...

**Comment by carusogabriel** [comment]
@malarzm Same as @lcobucci, 
> will fix something me and @alcaeus would like to see fixed

I look up at https://github.com/doctrine/mongodb-odm/pull/1757 and didn't get it :(

**Comment by alcaeus** [comment]
Yeah, looks like i wasn’t quite awake when reviewing the PR in question. elseif doesn’t make any sense when using return, so I’d keep the sniff enabled.

**Comment by malarzm** [comment]
All right, closing this 👍

**Pull request closed without merging**